libcalico-go/config/crd/crd.projectcalico.org_bgpconfigurations.yaml

```diff
@@ -68,6 +68,33 @@ spec:
                 description: 'LogSeverityScreen is the log severity above which logs
                   are sent to the stdout. [Default: INFO]'
                 type: string
+              nodeMeshMaxRestartTime:
+                description: Time to allow for software restart for node-to-mesh peerings.  When
+                  specified, this is configured as the graceful restart timeout.  When
+                  not specified, the BIRD default of 120s is used.
+                type: string
+              nodeMeshPassword:
+                description: Optional BGP password for full node-to-mesh peerings.
+                properties:
+                  secretKeyRef:
+                    description: Selects a key of a secret in the node pod's namespace.
+                    properties:
+                      key:
+                        description: The key of the secret to select from.  Must be
+                          a valid secret key.
+                        type: string
+                      name:
+                        description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                          TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
Is this TODO coming from our code?

Reply:
Yeah, I'll try to find why this got pulled in...

Reply:
Apparently it's from K8s from when we use their secret reference. If you trace it all back it finally gets to here: https://github.com/kubernetes/api/blob/master/core/v1/types.go#L5600-L5605